docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+Namespaces can only contain lowercase alphanumeric and dash characters.
+
+**Raises:**
+
+- GRPCError: An error is raised if an invalid namespace (for example, `""`) is provided.
+
+**Returns:**
+
+- [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): `True` if the provided namespace is available.
+
+```python {class="line-numbers linkable-line-numbers"}
+available = await cloud.get_organization_namespace_availability(public_namespace="my-cool-organization")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_organization_namespace_availability).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### UpdateOrganizationInviteAuthorizations
+
+Update the authorizations attached to an organization invite that has already been created.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `email` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Email of the user the invite was sent to.
+- `add_authorization` [(Optional[List[viam.proto.app.Authorization]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Authorization): Optional list of authorizations to add to the invite.
+- `remove_authorization` [(Optional[List[viam.proto.app.Authorization]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Authorization): Optional list of authorizations to remove from the invite.
+
+**Raises:**
+
+- GRPCError: An error is raised if no authorizations are passed or if an invalid combination of authorizations is passed (for example, an authorization to remove when the invite only contains one authorization).
+
+**Returns:**
+
+- [(OrganizationInvite)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.OrganizationInvite): The updated invitation.
+
+```python {class="line-numbers linkable-line-numbers"}
+update_invite = await cloud.update_organization_invite_authorizations(
+    email="notarealemail@viam.com",
+    remove_authorization=[
+        authorization_type="some type of auth",
+        authorization_id="identifier",
+        resource_type="abc",
+        resource_id="resource-identifier123",
+        identity_id="id12345",
+        organization_id="org_id_123"
+    ]
+)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.update_organization_invite_authorizations).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### NewRobot
+
+Create a new robot.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `name` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the new robot.
+- `location_id` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location under which to create the new robot.
```

Comment:
```suggestion
- `location_id` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location to create the new robot in.
``` 
(opt) Wording is a bit confusing

Reply:
I was trying to not commit the grammar faux pas of ending a sentence with a preposition, but maybe bending the rules is better for most readers here